src/main/java/teammates/sqllogic/api/Logic.java

```diff
@@ -620,6 +622,28 @@ public void adjustFeedbackSessionEmailStatusAfterUpdate(FeedbackSession session)
         feedbackSessionsLogic.adjustFeedbackSessionEmailStatusAfterUpdate(session);
     }
 
+    /**
+     * Gets the expected number of submissions for a feedback session.
+     *
+     * <br>Preconditions: <br>
+     * * All parameters are non-null.
+     */
+    public int getExpectedTotalSubmission(FeedbackSession fs) {
+        assert fs != null;
```

Comment:
i wonder if its better for us to handle this gracefully as exception rather than assert. 

since assertions are usually used for catastrophic failures and would crash the system